package.json

```diff
@@ -1,25 +1,27 @@
 {
   "name": "@c-frame/aframe-particle-system-component",
-  "version": "1.1.4",
+  "version": "1.1.5",
   "description": "Particle systems for A-Frame.",
-  "main": "index.js",
+  "browser": "index.js",
   "directories": {
     "example": "examples"
   },
   "devDependencies": {
-    "browserify": "^13.0.0",
-    "browserify-css": "^0.9.1",
-    "budo": "^8.2.2",
-    "randomcolor": "^0.4.4",
-    "semistandard": "^8.0.0",
-    "shelljs": "^0.7.0",
-    "shx": "^0.1.1",
-    "snazzy": "^4.0.0",
-    "webpack": "^1.13.0"
+    "@webpack-cli/generators": "^3.0.7",
+    "browserify": "^17.0.0",
+    "browserify-css": "^0.15.0",
+    "budo": "^11.8.4",
```

Comment:
You could also remove budo and use webpack serve instead.

Reply:
That would reduce the number of dependencies.

Reply:
You can also remove @webpack-cli/generators, it was just used to create the config, right?
See https://github.com/c-frame/aframe-cursor-teleport as a good example with webpack config for build and dev server.

Reply:
Done.